src/sdk/python/src/iotea/core/util/mqtt_client.py

```diff
@@ -66,6 +72,8 @@ def __prefix_topic_ns(self, topic):
         return MqttClient.prefix_topic_ns(topic, self.topic_ns)
 
     def __strip_topic_namespace(self, topic):
+        if (self.topic_ns is None):
```

Comment:
Remove parenthesis

```python
if self.topic_ns is None:
```

Reply:
Yes, thank you, Alex, I saw it, too. Will fix it when figuring out what breaks the integration tests.